sematic/resolvers/log_streamer.py

```diff
@@ -51,9 +52,21 @@
 _LAST_NON_EMPTY_DELTA_TEMPLATE = "{}.previous"
 _TERMINATION_CHAR = chr(ascii.EOT)  # EOT => End Of Transmission
 
+_INGESTION_ENABLED_MODE = "storage"
+_INGESTION_DISABLED_MODE = "off"
```

Comment:
In the check below, you only validate the value of "off"; "storage" is never checked.
Why not have a True/False flag instead?

Reply:
If we wind up wanting more ways to do ingestion than just "on"/"off", then we will want it expressed as a string. Technically the name of the config and the possible values are user-facing APIs, so making accommodations for future backwards compatibility makes sense so long as it is not too costly. All the non-user facing python logic is framed as bool for simplicity, it's literally just the var name and string that leave the door open.